### PR TITLE
Add Download button

### PR DESCRIPTION
**Pull request by manisacharya**
- Added new download button to download converted image

Related issue: https://github.com/KonradLinkowski/Voronoi/issues/2

Image:
image

### CODE REVIEW

**Comment by manisacharya** [comment]
Hey @KonradLinkowski, I tried this version, can you look at this version?

File: assets/css/style.css [hunk 1]
```diff
@@ -129,3 +129,17 @@ body {
   border-style: solid;
   border-color: #f9f9fb #f9f9fb #b8bec9 #b8bec9;
 }
+
+.button {
```
Comment (KonradLinkowski) [inline assets/css/style.css]: Can you change the class name to be more meaningful? Like `.download-button`.

File: assets/js/voronoi.js [hunk 2]
```diff
@@ -10,6 +10,14 @@ let mainImageData = getImageData(image);
 
 voronoi();
 
+var button = document.getElementById('btn-download');
```
Comment (KonradLinkowski) [inline assets/js/voronoi.js]: Don't use `var` please. You can use `const` in this case.

File: assets/js/voronoi.js [hunk 3]
```diff
@@ -10,6 +10,14 @@ let mainImageData = getImageData(image);
...
+
+button.addEventListener('click', function (e) {
```
Comment (KonradLinkowski) [inline assets/js/voronoi.js]: I would rather use arrow function here - it looks more readable

File: assets/js/voronoi.js [hunk 4]
```diff
@@ -10,6 +10,14 @@ let mainImageData = getImageData(image);
...
+  var image = canvas.toDataURL("image/png").replace("image/png", "image/octet-stream");
```
Comment (KonradLinkowski) [inline assets/js/voronoi.js]: `const`
Nice solution btw :)

**Review by KonradLinkowski — CHANGES_REQUESTED** [review]
Just remove the `var`s

**Comment by manisacharya** [comment]
@KonradLinkowski I have updated code as per your suggestions and also resolved merge conflict.

**Review by KonradLinkowski — APPROVED** [review]

**Pull request merged**